test-cases/README.md

```diff
@@ -37,6 +52,43 @@ npm run prettier
 
 Commit everything and open a new PR.
 
+## How to include a manual test case in the next release
+
+> Attention: all changes need to be manually pushed to master or to the release branch
+
+By default we don't execute all manual tests on each release, therefore if we know we have made a change to a component and a specific test should be executed to verify the change or prevent regressions we must manually add the test case to the next release.
+
+To do that the next version needs to added to the `targets`, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```

Comment:
```suggestion
To do that the next version needs to be added to the `targets`, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```

Reply:
Sorry @b1zzu , in my suggestion there was a mistake :+1: Once this is fixed, I am happy to lgtm and approve this one

Reply:
np, there was no rush to merge it